### PR TITLE
guide: FlutterFlow

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

- Adds `canonical` option the meta object to allow overriding the canonical URL
- Adds FlutterFlow guide

### CODE REVIEW

File: apps/docs/pages/guides/integrations/flutterflow.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+   5. Now, you must set the row you want to delete. Usually, this is done by finding a row in a table that matches the current row ID. To do so, click + Add Filter button inside the Matching Rows section.
+      1. Set the Field Name to the field that contains the IDs. Typically, this is the id column.
+      2. Set the Relation to Equal To because you want to find a row with the exact id.
+      3. Into the Value Source, you can select the From Variable and provide the id of the row you want to delete.
+
+![Delete data from Supabase](/docs/img/guides/integrations/flutterflow/delete.gif)
+
+## Resources
+
+You can find a more detailed
```
Comment (dannykng) [inline apps/docs/pages/guides/integrations/flutterflow.mdx:111]: @dshukertjr looks like this got cut off.

**Review by dannykng — APPROVED** [review]

**Comment by dannykng** [comment]
Also, the gifs are quite large in size. Thoughts on converting them to videos instead? Might help improve how quickly they load in the browser.

**Comment by dshukertjr** [comment]
> Also, the gifs are quite large in size. Thoughts on converting them to videos instead? Might help improve how quickly they load in the browser.

Sure, let me see what I can do!

**Comment by dshukertjr** [comment]
Converted the gif to videos, and they load much faster! And wow those gifs were large!

File: apps/docs/pages/guides/integrations/flutterflow.mdx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    src="/docs/videos/guides/integrations/flutterflow/delete.mp4"
+    type="video/mp4"
+    muted
+    playsInline
+  />
+</video>
+
+## Resources
+
+You can find more detailed guides on FlutterFlow’s docs
```
Comment (dannykng) [inline apps/docs/pages/guides/integrations/flutterflow.mdx:139]:
```suggestion
You can find more detailed guides on FlutterFlow’s docs.
```

**Review by dannykng — APPROVED** [review]
Looks great, thanks @dshukertjr!

**Pull request merged**